docs/source/user_guides/superset_guide.rst

```diff
@@ -0,0 +1,113 @@
+Superset guide
+---------------------
+
+Description
+~~~~~~~~~~~
+
+| The Dialog Flow Stats module can be used to obtain and visualize usage statistics for your service.
+| The module relies on several open source solutions that allow for data persistence and visualization
+
+* `Clickhouse <https://clickhouse.com/>`_ serves as an OLAP storage for data.
+* Batching and preprocessing data is based on `OpenTelemetry protocol <https://opentelemetry.io/docs/what-is-opentelemetry/>`_ and the `OpenTelemetry collector <https://opentelemetry.io/docs/collector/>`_.
+* Interactive visualization is powered by `Apache Superset <https://superset.apache.org/>`_.
+
+All the mentioned services are shipped as Docker containers, including a pre-built Superset image that ensures API compatibility.
+
+Collection procedure
+~~~~~~~~~~~~~~~~~~~~
+
+**Installation**
+
+.. code-block:: shell
+
+    pip install dff[stats]
+
+**Launching services**
+
+.. code-block:: shell
+    :linenos:
+
+    # clone the original repository to access the docker-compose file
+    git clone https://github.com/deeppavlov/dialog_flow_framework.git
+    # launch the required services
+    docker-compose up otelcol clickhouse dashboard
+
+**Collecting data**
+
+Collecting data is done by means of instrumenting your conversational service before you run it.
+DFF tutorials showcase all the necessary steps, needed to achieve that. We will run
+`one of those files <../tutorials/tutorials.stats.1_extractor_functions.py>`_ 
+in order to obtain sample data points to visualize.
+
+.. code-block:: shell
+
+    export DISABLE_INTERACTIVE_MODE=1 && python tutorials/stats/1_extractor_functions.py
+
+Displaying the data
+~~~~~~~~~~~~~~~~~~~
+
+In order to display the Superset dashboard, you should update the default configuration with the credentials of your database.
+The configuration can be optionally saved as a zip archive for inspection / debug.
+
+You can set most of the configuration options using a YAML file.
+
+.. code-block:: yaml
+    :linenos:
+
+    # config.yaml
+    db:
+        type: clickhousedb+connect
+        name: test
+        user: username
+        host: clickhouse
+        port: 8123
+        table: otel_logs
+
+The file can then be used to parametrize the configuration script.
+Password values will be prompted once you run the following command:
+
+.. code-block:: shell
+
+    dff.stats tutorials/stats/config.yaml -P -dP -U superset --outfile=config_artifact.zip
```

Comment:
I think, it would make more sense to demonstrate the yaml syntax than the file creation process; Especially given that we have a ready file in the 'tutorials' section.

Reply:
I think that file creation process also shows the syntax of the file, but OK.

I like the change from
```
.. code-block:: yaml
    :linenos:

    # config.yaml
```
to
```
.. code-block:: yaml
    :linenos:

    # tutorials/stats/config.yaml
```
but I think it needs more explanation.

Something like
```
You can set most of the configuration options using a YAML file.
You can find a configuration file that works with the default env variables
in the `tutorials/stats` directory:

.. code-block:: yaml
    :linenos:

    # tutorials/stats/config.yaml
```